src/EnergyPlus/HVACManager.cc

```diff
@@ -1105,47 +1107,46 @@ void SimHVAC(EnergyPlusData &state)
 
                     for (NodeIndex = 1; NodeIndex <= state.dataConvergeParams->ZoneInletConvergence(ZoneNum).NumInletNodes; ++NodeIndex) {
 
+                        auto &humRatInletNode = state.dataConvergeParams->ZoneInletConvergence(ZoneNum).InletNode(NodeIndex).HumidityRatio;
+                        auto &mdotInletNode = state.dataConvergeParams->ZoneInletConvergence(ZoneNum).InletNode(NodeIndex).MassFlowRate;
+                        auto &inletTemp = state.dataConvergeParams->ZoneInletConvergence(ZoneNum).InletNode(NodeIndex).Temperature;
+
```

Comment:
Adding local references to these variables in a few places to clean up how the code looks. I'm not stuck on the names, so let me know if you'd like to see something else.